2021-04-16 15:26:44,744 INFO    MainThread:31513 [wandb_setup.py:_flush():69] setting env: {}
2021-04-16 15:26:44,745 INFO    MainThread:31513 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-16 15:26:44,745 INFO    MainThread:31513 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210416_152644-3noxwmkl/logs/debug.log
2021-04-16 15:26:44,745 INFO    MainThread:31513 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210416_152644-3noxwmkl/logs/debug-internal.log
2021-04-16 15:26:44,746 INFO    MainThread:31513 [wandb_init.py:init():369] calling init triggers
2021-04-16 15:26:44,746 INFO    MainThread:31513 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-16 15:26:44,746 INFO    MainThread:31513 [wandb_init.py:init():418] starting backend
2021-04-16 15:26:44,746 INFO    MainThread:31513 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-16 15:26:44,800 INFO    MainThread:31513 [backend.py:ensure_launched():123] starting backend process...
2021-04-16 15:26:44,836 INFO    MainThread:31513 [backend.py:ensure_launched():127] started backend process with pid: 31577
2021-04-16 15:26:44,838 INFO    MainThread:31513 [wandb_init.py:init():423] backend started and connected
2021-04-16 15:26:44,842 INFO    MainThread:31513 [wandb_init.py:init():465] updated telemetry
2021-04-16 15:26:44,844 INFO    MainThread:31513 [wandb_init.py:init():484] communicating current version
2021-04-16 15:26:46,694 INFO    MainThread:31513 [wandb_init.py:init():489] got version response 
2021-04-16 15:26:46,695 INFO    MainThread:31513 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-16 15:26:46,857 INFO    MainThread:31513 [wandb_init.py:init():522] starting run threads in backend
2021-04-16 15:26:50,652 INFO    MainThread:31513 [wandb_run.py:_console_start():1524] atexit reg
2021-04-16 15:26:50,654 INFO    MainThread:31513 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-16 15:26:50,655 INFO    MainThread:31513 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-16 15:26:50,657 INFO    MainThread:31513 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-16 15:26:50,658 INFO    MainThread:31513 [wandb_init.py:init():546] run started, returning control to user process
2021-04-16 15:46:47,061 INFO    MainThread:31513 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-16 15:46:47,062 INFO    MainThread:31513 [wandb_run.py:_restore():1466] restore
2021-04-16 15:46:49,219 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 30545
  total_bytes: 30545
}

2021-04-16 15:46:49,388 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 30545
  total_bytes: 30545
}

2021-04-16 15:46:49,823 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 30545
  total_bytes: 36242
}

2021-04-16 15:46:50,002 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 30702
  total_bytes: 36242
}

2021-04-16 15:46:50,104 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 36242
  total_bytes: 36242
}

2021-04-16 15:46:50,207 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 36242
  total_bytes: 36242
}

2021-04-16 15:46:50,310 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 36242
  total_bytes: 36242
}

2021-04-16 15:46:50,412 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 36242
  total_bytes: 36242
}

2021-04-16 15:46:50,515 INFO    MainThread:31513 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 36242
  total_bytes: 36242
}

2021-04-16 15:46:51,960 INFO    MainThread:31513 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-16 15:46:51,960 INFO    MainThread:31513 [wandb_run.py:_show_files():1838] logging synced files
